content/en/docs/event-sources/enable-event-sources.md

```diff
@@ -50,4 +50,4 @@ This logic does not apply when Falco is reproducing events from a capture file.
 
 Since [version 3.0.0 of the plugin API](/docs/reference/plugins/plugin-api-reference/#plugin-api-versioning), released starting from Falco [0.35.0](/blog/falco-0-35-0/), plugins with event sourcing capability have the possibility of producing system events just like the [Falco Drivers](/docs/event-sources/drivers/). Falco runs plugins of this kind as regular [plugins with event sourcing capability](/docs/reference/plugins/plugin-api-reference/#event-sourcing-capability-api) loaded through [Falco's configuration](/docs/reference/daemon/config-options/), and would interpret their supported event source as `syscall`.
 
-When the `syscall` source is enabled, by default Falco always opens its event stream by leveraging one of the configured Falco Drivers. When a plugin implementing the `syscall` event source is loaded through the Falco configuration, users can specify the `--nodriver` option to instruct Falco about opening the `syscall` event stream by using the plugin instead of the drivers. The `--nodriver` option is used in mutual exclusion with the other driver selection options (e.g. `-g`, `modern-bpf`, etc...).
+When the `syscall` source is enabled, by default Falco always opens its event stream by leveraging one of the configured Falco Drivers. When a plugin implementing the `syscall` event source is loaded through the Falco configuration, users can specify the `-o engine.kind=nodriver` option to instruct Falco about opening the `syscall` event stream by using the plugin instead of the other engines.
```

Comment:
```suggestion
By default, when the `syscall` source is enabled, Falco opens its event stream by leveraging one of the configured Falco Drivers. However, if a plugin that implements the `syscall` event source is loaded through the Falco configuration, users can specify `-o engine.kind=nodriver` to instruct Falco to use the plugin instead of a driver for the `syscall` source.
```

Trying to improve clarity and smoothness.

Reply:
done :)